### PR TITLE
Add support for GetIncludePaths()

### PR DESCRIPTION
**Pull request by Krishna-13-cyber**

### CODE REVIEW

File: include/clang/Interpreter/CppInterOp.h [hunk 1]
```diff
@@ -439,6 +439,8 @@ namespace Cpp {
   /// GetResourceDir() function.
   void AddIncludePath(const char *dir);
 
+  std::vector <std::string> GetIncludePath(const char *dir);
```
Comment (vgvassilev) [inline include/clang/Interpreter/CppInterOp.h:442]:
```suggestion
  void GetIncludePath(const char *dir, std::vector <std::string>& Paths);
```

We should use that signature as it is more efficient. Can you also add tests for this?

**Comment by vgvassilev** [comment]
@Krishna-13-cyber can you rebase your PR?

**Comment by Krishna-13-cyber** [comment]
Hi @vgvassilev 
Yes I tried to rebase but there are set of conflicts, I will get this done shortly.
Thanks.

**Comment by Krishna-13-cyber** [comment]
Closing this due to some excessive conflicts on my side.

**Pull request closed without merging**